coral/src/app/components/Form.tsx

```diff
@@ -354,12 +354,6 @@ function _MultiSelect<T extends FieldValues, FieldValue>({
             }}
             valid={error === undefined}
             helperText={error}
-            // If we do not explicitly pass value to the component
```

Comment:
Haha nice, I remember seeing this and wanting to open a PR to remove it a few times, but never got around to it

Reply:
Sometimes a nice little clean up PR is the best way to get rid of this small things :D It never fits in the PR you're working on when you see it but is too small to really feel good making a PR out of it :D